### DISCUSSION TITLE
Frontend Flutter Developer – Senior to Midlevel - Docu tools

### DISCUSSION THREAD

**Discussion by renanstig** [Flutterando/forum#71] [Vagas e Oportunidades]
Estamos em busca de um dev flutter para iniciar o desenvolvimento do nosso aplicativo da empresa em Web e Mobile (Android)

A vaga é full remote, é precisamos de uma pessoa comprometida para que possamos criar um relacionamento de longa duração. A empresa é a [docu tools](https://www.docu-tools.com/en/), ela é baseada em Viena, na Áustria. Atualmente já temos dois brasileiros trabalhando remoto do Brasil, portanto temos uma certa experiência com isso. 

Para mais informações, por favor leia o arquivo em anexo. 

[docu tools flutter.pdf](https://github.com/Flutterando/forum/files/9821305/docu.tools.flutter.pdf)

**Comment by FredericoWP** [comment]
> Estamos em busca de um dev flutter para iniciar o desenvolvimento do nosso aplicativo da empresa em Web e Mobile (Android)
> 
> A vaga é full remote, é precisamos de uma pessoa comprometida para que possamos criar um relacionamento de longa duração. A empresa é a [docu tools](https://www.docu-tools.com/en/), ela é baseada em Viena, na Áustria. Atualmente já temos dois brasileiros trabalhando remoto do Brasil, portanto temos uma certa experiência com isso.
> 
> Para mais informações, por favor leia o arquivo em anexo.
> 
> [docu tools flutter.pdf](https://github.com/Flutterando/forum/files/9821305/docu.tools.flutter.pdf)

Olá @renanstig , esta vaga já foi preenchida ?
Nesta documentação não vi o nível de conversão em Inglês exigido. 
Caso não tenha preenchido a vaga, poderia nos informar o nível ?

**Reply by renanstig** [reply]
Ola @FredericoWP , 
Me desculpe por não informar o nível de inglês. A vaga está em aberto sim. A gente está aceitando inglês intermediário. Você precisa ser capaz de: participar de reuniões(grooming, standup, retrospectiva), entender requisitos e apresentar o projeto, se necessário. 
Se você estiver aprendendo inglês conta muito pra gente. 
Qualquer outra dúvida,

**Reply by FredericoWP** [reply]
Ok, obrigado pelo retorno!

**Reply by renanstig** [reply]
Qualquer outra dúvida, só mandar mensagem e se tiver interesse, me envia o CV 😃

**Reply by FredericoWP** [reply]
> Qualquer outra dúvida, só mandar mensagem e se tiver interesse, me envia o CV 😃

Ok, até me responderam pelo email que enviei o cv, mas se possivel gostaria de tirar uma dúvida contigo 😅.
Pode me informar a melhor forma de contactar você no pv ? Ou se puder me ta um toque no telegram ou Whatsapp 5531998734481.
Agradeço desde já!
